ci/macOS.yml

```diff
@@ -40,13 +40,11 @@ jobs:
       condition: and(succeeded(), eq(variables.is_release, 'true'))
     - template: get-local-canton.yml
     - template: clean-up.yml
-    # Do not build m1 on releases
-    - ${{ if or(eq(parameters.name, 'macos'), eq(variables.is_release, 'false')) }}:
-      - template: build-unix.yml
-        parameters:
-          release_tag: $(release_tag)
-          name: ${{parameters.name}}
-          is_release: variables.is_release
+    - template: build-unix.yml
```

Comment:
Turns out this doesn't quite work as I'd thought. The `${{}}` expressions denote a compile-time expression whereas the `is_release` variable is a runtime variable, which means that by the time of compilation it is not set and is thus not equal to 'false'. So that meant `m1` builds never ran.

Unfortunately I don't know of a way to disable an entire template using run-time variables, so instead this PR disables all the steps individually.